packages/account-v2/src/containers/TradingAssessmentForm/TradingAssessmentForm.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import { Form, Formik } from 'formik';
+import { FormSubHeader } from '../../components/FormSubHeader';
+import { getTradingAssessmentQuestions } from '../../constants/tradingAssessmentQuestions';
+import { RiskToleranceWarningModal } from './RiskToleranceWarningModal';
+import { TestWarningModal } from './TestWarningModal';
+import { TradingAssessmentContainer } from './TradingAssessmentContainer';
+
+export const TradingAssessmentForm = () => {
+    const [shouldShowRiskTolerence, setShouldShowRiskTolerence] = useState(false);
+    const [shouldShowTestWarningModal, setShouldShowTestWarningModal] = useState(false);
+    const tradingAssessmentQuestions = getTradingAssessmentQuestions();
+
```

Comment:
shouldnt we set Modal.setAppElement(ACCOUNT_MODAL_REF); before opening the modals?

Reply:
Adding it to the popup components 🙏